index.htm

```diff
@@ -24,15 +24,74 @@
     <a href="#" class="button filled-primary-button">Sign-up</a>
   </nav>
   <div class="wrapper">
-      <div id="hero-img">
-        <div id="hero-text">
-          <h1>We Make Your Home Alive</h1>
+    <div id="hero-img">
+      <div id="hero-text">
+        <h1>We Make Your Home Alive</h1>
+      </div>
+      <div id="center-button">
+        <a href="#" class="button filled-primary-button">Shop</a>
+        <a href="#" class="button outlined-secondary-button">Blog</a>
+      </div>
+    </div>
+    <h2 id="gallery-heading">Our Loved Ones</h2>
+    <div id="gallery-container">
+      <div class="gallery-row">
+        <div class="gallery-column">
+          <img src="./res/img/1.jpeg" width="100%">
+          <div class="common-name gallery-box">Common Name</div>
+          <div class="scientific-name" id="scientific-name-color-1">Scientific Name</div>
+        </div>
+        <div class="gallery-column">
+          <img src="./res/img/2.jpeg" width="100%">
+          <div class="common-name gallery-box">Common Name</div>
+          <div class="scientific-name" id="scientific-name-color-1">Scientific Name</div>
+        </div>
+        <div class="gallery-column">
+          <img src="./res/img/3.jpeg" width="100%">
+          <div class="medium-difficulty gallery-box">Medium Difficulty</div>
```

Comment:
the name box seems arbitrary, since the container is a box too (currently named gallery-column)- i would name that a gallery-label.